src/Hazelcast.Net/Clustering/ClusterMembers.cs

```diff
@@ -932,9 +932,10 @@ internal ICollection<MemberInfo> FilterMembers(ICollection<MemberInfo> members)
                     => !subsetMembersIds.Contains(m.Id)).ToList();
 
                 _logger.LogDebug("Filtered members from member view: {RemovedCount} removed, {FilteredCount} filtered," +
-                                 " {MembersCount} total. Removed Member Ids :[{Members}]",
+                                 " {MembersCount} total. Remaining members: [{NewMembers}] Removed Member Ids :[{Members}]",
```

Comment:
```suggestion
                                 " {MembersCount} total. Remaining members: [{NewMembers}]. Removed Member Ids: [{Members}]",
```
Massive nitpick, don't bother unless you make other changes at the same time 😄

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/907/commits/7a6a44c4fe4d14654f458efd48f31189cbde1e19